play-interop-test-scala/src/test/scala/akka/grpc/play/api/specs2/ServerGrpcClient.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.play.api.specs2
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.grpc.internal.{ AkkaGrpcClient, AkkaGrpcClientFactory }
+import akka.stream.Materializer
+import com.typesafe.config.{ Config, ConfigFactory, ConfigValueFactory }
+import play.api.test.{ NewWithServer, ServerEndpoint }
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ Await, ExecutionContext }
+
+/**
+ * Helpers to test gRPC clients with Play using Specs2.
+ */
+trait ServerGrpcClient { this: NewWithServer =>
+
+  /**
+   * Create a gRPC client to connect to the currently running test server.
+   * @tparam T The type of client to create.
+   * @return
+   */
+  protected def withGrpcClient[T <: AkkaGrpcClient]: WithGrpcClient[T] = new WithGrpcClient[T]
```

Comment:
Return a class with an `apply[U]` method so that we can explicitly provide a `T` parameter but have the compiler infer the `U` parameter.